### PR TITLE
Add a command for velero backup details

### PR DESCRIPTION
**Pull request by apurvabhide17**
Version(s):
OADP 1.4.0
OCP 4.12 → OCP 4.16

Issue:
[OADP-1444](https://issues.redhat.com/browse/OADP-1444)

Link to docs preview: https://77086--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#backup-cr-remains-in-progress_oadp-troubleshooting 

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by apurvabhide17** [comment]
Thank you, @tmalove for reviewing it.

File: modules/oadp-backup-restore-cr-issues.adoc [hunk 1]
```diff
@@ -48,10 +48,17 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
...
-You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded  files to object storage.
+You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded files to object storage.
 
 . Create a new `Backup` CR.
 
+. View the Velero backup details
++
+[source,terminal]
+----
+$ velero backup describe --details
```
Comment (PrasadJoshi12) [inline modules/oadp-backup-restore-cr-issues.adoc:59]:
```suggestion
$ velero backup describe <backup-name> --details
```

Comment (apurvabhide17) [reply modules/oadp-backup-restore-cr-issues.adoc]: @PrasadJoshi12, thanks, I've added that.

**Comment by apurvabhide17** [comment]
@tmalove, added a minor change, could you review and let me know if it looks good? 
Thanks!
/label peer-review-needed

**Comment by GroceryBoyJr** [comment]
@apurvabhide17 tmalove is off peer review this week, I'll pick up your PR for peer review now.

File: modules/oadp-backup-restore-cr-issues.adoc [hunk 2]
```diff
@@ -48,10 +48,17 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
...
 ----
 +
-You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded  files to object storage.
+You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded files to object storage.
 
 . Create a new `Backup` CR.
 
+. View the Velero backup details
++
+[subs="+quotes"]
```
Comment (GroceryBoyJr) [inline modules/oadp-backup-restore-cr-issues.adoc:57]:
```suggestion
[source,terminal]]
```

**Review by GroceryBoyJr — COMMENTED** [review]
Found one issue, otherwise, LGTM.  You should be able to go on to merge review after correcting this one issue.

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/oadp-backup-restore-cr-issues.adoc (hunk 2 above)
Comment (apurvabhide17) [reply modules/oadp-backup-restore-cr-issues.adoc:57]: @GroceryBoyJr, the replaceable syntax wasn't working when added with [source, terminal]. Isn't it recommended here to use [subs="+quotes"]?

**Comment by apurvabhide17** [comment]
Hello @PrasadJoshi12, 
Could you share a example backup file name, which I can add to the command? This helps the reader to look for similar extension files while executing that command?
Thanks!

File: modules/oadp-backup-restore-cr-issues.adoc [hunk 3]
```diff
@@ -48,10 +48,17 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
...
-You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded  files to object storage.
+You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded files to object storage.
 
 . Create a new `Backup` CR.
 
+. View the Velero backup details
++
+[source,terminal, subs="+quotes"]
+----
+$ velero backup describe _<backup-name>_ --details
```
Comment (mburke5678) [inline modules/oadp-backup-restore-cr-issues.adoc:59]: I don't think we use the italicized variables in OCP docs. I see a few examples, but not many.

Comment (mburke5678) [reply modules/oadp-backup-restore-cr-issues.adoc:59]: I think that the discrepancy with the italics isn't sufficient to hold up the merge. Please consider this matter for a follow-up PR.

**Pull request merged**